pkg/printers/internalversion/printers.go

```diff
@@ -428,6 +428,7 @@ func AddHandlers(h printers.PrintHandler) {
 		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
 		{Name: "Holder", Type: "string", Description: coordinationv1.LeaseSpec{}.SwaggerDoc()["holderIdentity"]},
 		{Name: "Age", Type: "string", Description: metav1.ObjectMeta{}.SwaggerDoc()["creationTimestamp"]},
+		{Name: "RenewTime", Type: "string", Description: coordinationv1.LeaseSpec{}.SwaggerDoc()["renewTime"]},
```

Comment:
I believe that this resource is managed by sig-scalability and before making this decision, I'd recommend asking opinions from them (also need a issue to track discussions and the conclusion).

Reply:
thanks for this.
open issue to track: https://github.com/kubernetes/kubectl/issues/1669